frontend/common/Input/index.ts

```diff
@@ -30,13 +30,11 @@ export default class Input extends Container {
   }
 
   render() {
-    return `
-            <input
-            ${this.props.type ? `type=${this.props.type}` : ""}
-            class="Input" id="input-${this.ID}"
-            value="${this.props.value ?? ""}"
-            placeholder="${this.props.placeholder ?? ""}"></input>
-        `;
+    return `<input
+      ${this.props.type ? `type=${this.props.type}` : ""}
+      class="Input" id="input-${this.ID}"
+      value="${this.props.value ?? ""}"
```

Comment:
```suggestion
      type="${this.props.type ?? "text"}"
      class="Input" id="input-${this.ID}"
      value="${this.props.value ?? ""}"
```
type은 value처럼 표현할 수는 없으려나요.
default type인 text로 설정할 수도 있겠네요.